namer/rancher/src/main/scala/io/buoyant/namer/rancher/RancherClient.scala

```diff
@@ -0,0 +1,140 @@
+package io.buoyant.namer.rancher
+
+import com.fasterxml.jackson.databind.{ObjectMapper, PropertyNamingStrategy}
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
```

Comment:
linkerd import style is to group all imports together (no blank lines between imports) and sort them alphabetically.

Reply:
Are there any rules for how many imports you can do from each package before switching to `xyz._`?

Right now I explicitly name 8 different classes from `com.twitter.utils` because I wanted to actually know where each class came from as I was figuring out how stuff worked. But I guess I could just as well import `com.twitter.utils._`

Reply:
Personally, I usually wildcard it after about 5 imports in the same package.